odoo/fields.py

```diff
@@ -1246,9 +1250,12 @@ def convert_to_export(self, value, record):
 
 
 class Monetary(Field):
-    """ The decimal precision and currency symbol are taken from the attribute
+    """ Encapsulates a :class:`float` expressed in a given :class:`res_currency<odoo.addons.base.models.res_currency.Currency>`.
```

Comment:
```suggestion
    """ Encapsulates a :class:`float` expressed in a given :class:`res_currency
    <odoo.addons.base.models.res_currency.Currency>`.
```
No auto-generated link ☃️

Reply:
It's normal, res_currency doc isn't auto generated anywhere, but when we do, the link will be generated :stuck_out_tongue:.

Reply:
Still, break the line as suggested :p